### PR TITLE
Add Injury to stat file

### PR DESCRIPTION
**Pull request by dotcypress**
This PR adds `Injury` information to `StatFile`.

### CODE REVIEW

File: stat_file.go [hunk 1]
```diff
@@ -113,6 +113,7 @@ type StatFile struct {
 	Play                     []*StatFilePlay
 	PlayStat                 []StatFilePlayStat
 	PlayStatNullified        []StatFilePlayStat
+	Injury                   []StatFilePlayStat
```
Comment (ccbrown) [inline stat_file.go:116]: Should this be a different type?

Comment (dotcypress) [reply stat_file.go]: Yes, we lost `ReturnStatus` here, fixed.

File: stat_file.go [hunk 2]
```diff
@@ -200,10 +209,21 @@ func (f *StatFile) Update(update *StatFile) {
...
+					n += 1
+				}
+			}
+			f.Injury = f.Injury[:n]
+			f.Injury = append(f.Injury, update.Injury...)
 		} else {
 			f.Play = append(f.Play, p)
 			f.PlayStat = append(f.PlayStat, update.PlayStat...)
 			f.PlayStatNullified = append(f.PlayStatNullified, update.PlayStatNullified...)
+			f.Injury = append(f.Injury, update.Injury...)
```
Comment (ccbrown) [inline stat_file.go:226]: We should also sort these below, where we sort stats.

**Review by ccbrown — CHANGES_REQUESTED** [review]
See sorting comment.

**Review by ccbrown — APPROVED** [review]

**Pull request merged**